data/map planets.txt

```diff
@@ -118,20 +120,20 @@ planet Inova
 	"required reputation" -1000
 
 planet Prodos
-	attributes border station
+	attributes periphery station
 	landscape land/station10
-	description "Prodos station was one of the three stations built during the Caravan Era and was a symbol of progress for this region of space, the pirate assault of 2873 left the Station partially damaged and many of its residents escaping to Solace."
+	description "Prodos Station was one of the three stations built during the Caravan Era, a symbol of progress for this region of space. The pirate assault of 2873 left the station partially damaged, and many of its residents were evacuated to Solace."
```

Comment:
I would make the date much later, like in the late 2900s, to reflect the Republic's retreat from this area and the formation of the Southern Mutual Defence Pact.

Also, have you considered giving a headquarters to the SMDP? I would put it on Glaze, but I had a mission chain (that I once thought about pulling to endless-sky/endless-sky) that stated that it was on Solace, because of a particularly nasty combined raid by raiders from Greenrock and Thule.